stdlib/mexpr/eval.mc

```diff
@@ -838,30 +861,22 @@ let concatAst = concat_
 utest eval concatAst
 with seq_ [int_ 1, int_ 2, int_ 3, int_ 4, int_ 5, int_ 6] in
 
--- length [1, 2, 3] = 3
+-- length [1, 2, 3] -> 3
 let lengthAst = length_ (seq_ [int_ 1, int_ 2, int_ 3]) in
 utest eval lengthAst with int_ 3 in
 
--- tail [1, 2, 3] = [2, 3]
-let tailAst = tail_ (seq_ [int_ 1, int_ 2, int_ 3]) in
-utest eval tailAst with seq_ [int_ 2, int_ 3] in
-
--- head [1, 2, 3] = 1
-let headAst = head_ (seq_ [int_ 1, int_ 2, int_ 3]) in
-utest eval headAst with int_ 1 in
-
--- null [1, 2, 3] = false
-let nullAst = null_ (seq_ [int_ 1, int_ 2, int_ 3]) in
-utest eval nullAst with false_ in
-
--- null [] = true
-let nullAst = null_ (seq_ []) in
-utest eval nullAst with true_ in
-
--- reverse [1, 2, 3] = [3, 2, 1]
+-- reverse [1, 2, 3] -> [3, 2, 1]
 let reverseAst = reverse_ (seq_ [int_ 1, int_ 2, int_ 3]) in
 utest eval reverseAst with seq_ [int_ 3, int_ 2, int_ 1] in
 
+-- splitAt [1,4,2,3] 2 -> ([1,4],[2,3])
+let splitAtAst = splitat_ (seq_ [int_ 1, int_ 4, int_ 2, int_ 3]) (int_ 2) in
+-- utest eval splitAtAst
```

Comment:
Why is this test commented out? I initially thought it was because of the "`splitAt` is not tested since we cannot yet compile tuple projection" comment, but this is the interpreter, no?

Reply:
Fixed.